<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>612</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MFactory</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>561</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Connect</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>461</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Login</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>USER</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>PASSWORD</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="USER_text">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>361</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Password_text">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>361</width>
         <height>41</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="DeleteEmployee_button_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>260</y>
        <width>80</width>
        <height>37</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Employees">
     <attribute name="title">
      <string>Employees</string>
     </attribute>
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>961</width>
        <height>511</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Edit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>961</width>
         <height>425</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Edit</string>
       </attribute>
       <widget class="QPushButton" name="AddEmployeeButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>370</y>
          <width>80</width>
          <height>37</height>
         </rect>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
       <widget class="QPushButton" name="DeleteEmployee_button">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>370</y>
          <width>80</width>
          <height>37</height>
         </rect>
        </property>
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>390</y>
          <width>165</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Status : not conncected </string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>0</y>
          <width>331</width>
          <height>361</height>
         </rect>
        </property>
        <property name="title">
         <string>Delete Employee</string>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>CIN</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="CIN_Del_Text">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>160</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>381</width>
          <height>341</height>
         </rect>
        </property>
        <property name="title">
         <string>Add | Updtae Employee</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>CIN</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Full Name</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>150</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>200</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Address</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>250</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
        <widget class="QTextEdit" name="CIN_Employee_text">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>251</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="Name_Employee_text">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>100</y>
           <width>251</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="Age_Employee_text">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>150</y>
           <width>251</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="Address_Employee_text">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>200</y>
           <width>251</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QTextEdit" name="Email_Employee_text">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>250</y>
           <width>251</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>961</width>
         <height>425</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Display</string>
       </attribute>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>260</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>711</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="DeleteEmployee">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>0</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: rgb(0,0,0,0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="AddEmployee">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="statusTip">
         <string comment="Add Employee" extracomment="Add Employee"/>
        </property>
        <property name="styleSheet">
         <string notr="true">border: rgb(0,0,0,0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>0</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton:pressed{
	
	background-color: rgb(255, 88, 11);
}</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>140</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Full Name</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableEmployees">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>711</width>
          <height>391</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>100</horstretch>
          <verstretch>111</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string comment="Employee Table"/>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>CIN</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Full Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Age</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Address</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Email</string>
         </property>
        </column>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>50</y>
          <width>231</width>
          <height>261</height>
         </rect>
        </property>
        <property name="title">
         <string>Sort by</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>100</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QPushButton" name="UpdateEmployee">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>0</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: rgb(0,0,0,0)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>220</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>180</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>51</y>
          <width>721</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Posts">
     <property name="accessibleName">
      <string>Posts</string>
     </property>
     <attribute name="title">
      <string>Posts</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>711</width>
        <height>391</height>
       </rect>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>CIN</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Salary</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Benefits</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Hours worked</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Job</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>90</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton:pressed{
	
	background-color: rgb(255, 88, 11);
}</string>
      </property>
      <property name="text">
       <string>Statisitics</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DeletePosts">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: rgb(0,0,0,0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdatePosts">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: rgb(0,0,0,0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="AddPosts">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: rgb(0,0,0,0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>tableWidget_2</zorder>
     <zorder>pushButton_6</zorder>
     <zorder>UpdatePosts</zorder>
     <zorder>DeletePosts</zorder>
     <zorder>AddPosts</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDoc">
   <property name="text">
    <string>Doc</string>
   </property>
  </action>
  <action name="actionResources">
   <property name="text">
    <string>Resources</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
